### DISCUSSION TITLE
Zustand Error: Missing getServerSnapshot

### DISCUSSION THREAD

**Discussion by Gw3i** [neuefische/questions#66] [Web] [answered]
Good morning everyone,
I'm getting an error when I want to use zustand in my form page: <br>
**Server Error
Error: Missing getServerSnapshot, which is required for server-rendered content. Will revert to client rendering.** <br>
I've googled the error, but found nothing helpful.
<br>
<img width="1428" alt="Screenshot 2022-08-16 at 10 17 09" src="https://user-images.githubusercontent.com/101993818/184832342-9793e77c-8821-4b31-9c2b-1eaed1a68a51.png">

When I comment my state out, everything works fine.
 
<img width="1424" alt="Screenshot 2022-08-16 at 10 17 53" src="https://user-images.githubusercontent.com/101993818/184832385-ff7c8b9c-1fa3-4181-a976-c5f0f47dfe37.png">

Here are some screenshots:
<img width="1000" alt="Screenshot 2022-08-16 at 10 15 17" src="https://user-images.githubusercontent.com/101993818/184832713-293fc8b1-f9fe-4315-a130-3c2831430f7c.png">
<img width="991" alt="Screenshot 2022-08-16 at 10 15 24" src="https://user-images.githubusercontent.com/101993818/184832475-3d411e6d-3d9a-49b7-81b5-9a6643808182.png">
<img width="1158" alt="Screenshot 2022-08-16 at 10 15 53" src="https://user-images.githubusercontent.com/101993818/184832490-3f28b205-8a46-4cd6-8280-cadc77adb9c9.png">
<img width="1419" alt="Screenshot 2022-08-16 at 10 22 38" src="https://user-images.githubusercontent.com/101993818/184833306-3c0032ff-309a-4607-ad49-f4af61f9f7d5.png">

**Comment by alexclaes** [comment, marked as answer]
Hi Vlady,

please provide a link to your branch / pull request next time ;-) 

You are not using the `useStore` hook you wrote under `hooks/useStore.js`. There is something wrong with the import of the hook.

Please compare your code to the example in the project template:
https://github.com/neuefische/capstone-next-template/blob/main/src/pages/index.js

**Reply by Gw3i** [reply]
That helped 🎉 Thank you!

**Comment by pixelass** [comment]
Versuch es mal mit der aktuellen Beta: https://www.npmjs.com/package/zustand/v/4.0.0-beta.3

`npm i zustand@4.0.0-beta.3`
Das war wohl ein bug der gerade behoben wird.
